Sources/SKCore/Configuration.swift

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import Utility
+import PackageModel
+
+public protocol Configuration {
```

Comment:
What value is `Configuration` providing?  Is the idea to leave room for non-build configuration items?  I suggest dropping this for now. We can easily add it back if we need it.